{{cookiecutter.python_name}}/jupyter_server_test_config.py

```diff
@@ -0,0 +1,17 @@
+"""Server configuration for integration tests.
+
+!! Never use this configuration in production because it
+opens the server to the world and provide access to JupyterLab
+JavaScript objects through the global window variable.
+"""
+from tempfile import mkdtemp
```

Comment:
real estate in the root folder is precious: recommend putting this in the tests folder